src/main/java/seedu/address/logic/commands/HomeCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+import seedu.address.ui.SecondaryPaneState;
+
+/**
+ * Command to return the secondary pane to the home page.
+ */
+public class HomeCommand extends Command {
+
+    public static final String COMMAND_WORD = "home";
+    public static final String MESSAGE_SUCCESS = "Displaying the home page. Welome back!";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        return new CommandResult(MESSAGE_SUCCESS, false, false, SecondaryPaneState.WELCOME);
```

Comment:
perhaps assign the `false` to a static variable to make this more readable? or there could be another `CommandResult` constructor so that there is no need to specify the other values

Reply:
Hmm alright, I think another Constructor will be cleaner. Will update!